ec2/sc-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -326,20 +326,31 @@ Resources:
         TagInstance:
           files:
             /opt/sage/bin/apply_name_tag.sh:
-              source: "https://raw.githubusercontent.com/Sage-Bionetworks/service-catalog-utils/v1.0.5/linux/opt/sage/bin/apply_name_tag.sh"
+              source: "https://raw.githubusercontent.com/Sage-Bionetworks/service-catalog-utils/v1.0.6/linux/opt/sage/bin/apply_name_tag.sh"
               mode: "00744"
               owner: "root"
               group: "root"
           commands:
             01_name_tag:
               command: "/bin/bash /opt/sage/bin/apply_name_tag.sh"
+        WriteApacheConf:
+          files:
+            /opt/sage/bin/apache_conf_rewrite.sh:
+              source: "https://raw.githubusercontent.com/Sage-Bionetworks/service-catalog-utils/v1.0.6/linux/opt/sage/bin/apache_conf_rewrite.sh"
+              mode: "00744"
+              owner: "root"
+              group: "root"
+          commands:
+            01_rewrite_apache_conf:
+              command: "/bin/bash /opt/sage/bin/apache_conf_rewrite.sh"
```

Comment:
Note that you need to do more than just defining this `WriteApacheConf` block. You need to actually get things to use it. Notice how `TagInstance` defined above is used: it is one of the steps included in the `SetTags` configSet. That configSet is passed to the cfn-init script as a parameter in the contents of the file cfn-auto-reloader.conf and in `UserData`.

Reply:
OK, interesting... I did not know.. I'll add a new configSet.